packages/documentation/.storybook/preview.js

```diff
@@ -30,5 +30,10 @@ export const parameters = {
       color: /(background|color)$/i,
       date: /Date$/
     }
+  },
+  docs: {
+    source: {
+      excludeDecorators: true
```

Comment:
What effect does that have?

Reply:
If we decorate (wrap) the output code with some html elements, these elements will show up in the code panel. If we exclude decorators in source component, they will not not.
This has nothing to do with angular decorators ;)